go.mod

```diff
@@ -19,23 +19,23 @@ require (
 	github.com/stretchr/testify v1.7.0
 	go.uber.org/zap v1.19.1
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
-	k8s.io/api v0.21.4
-	k8s.io/apimachinery v0.21.4
-	k8s.io/client-go v0.21.4
-	knative.dev/eventing v0.27.1
+	k8s.io/api v0.22.5
+	k8s.io/apimachinery v0.22.5
+	k8s.io/client-go v0.22.5
+	knative.dev/eventing v0.29.0
 	knative.dev/eventing-kafka v0.27.0
```

Comment:
any reason these got not bumped, while other eventing libs got bumped?

Reply:
This is weird didnt touch eventing during upgrading deps.